### DISCUSSION TITLE
Prompt Editor

### DISCUSSION THREAD

**Discussion by qunash** [qunash/chatgpt-advanced#44] [Announcements]
By popular demand, there's now a **Prompt Editor** on extension's options page.
Here you can add your own prompts/instructions, which can then be selected from the extension's toolbar.

![image](https://user-images.githubusercontent.com/3750161/214865176-8fffd153-ef62-47e5-b056-3c16ec9356b7.png)
![image](https://user-images.githubusercontent.com/3750161/214865244-8b25cc5f-eee3-4420-8189-49e702ea8851.png)

**Comment by WIWI84** [comment]
Hello
